src/main/python/salento/reports/map_computation/anomaly_score.py

```diff
@@ -30,18 +30,57 @@
 import json
 import argparse
 import gc
+import heapq
 # project imports
 import metric
 import data_parser
 
 
+class FilterTraces(object):
+    """ Implement different filtering options"""
+
+    @staticmethod
+    def filter_using_location(data, use_location=True):
+        """
+        :param data: input data list of dict, with the following keys
+                ['Index List', // bug index
```

Comment:
Please document any additional invariants that hold on these lists: e.g, the size of the index list, location list, call/event list, prob. vector is the same)?